MBBSEmu.Tests/ExportedModules/Majorbbs/strtol_Tests.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections.Generic;
+using System.Text;
+using Xunit;
+
+namespace MBBSEmu.Tests.ExportedModules.Majorbbs
+{
+    public class strtol_Tests : ExportedModuleTestBase
+    {
+        private const int STRTOL_ORDINAL = 586;
+
+        [Theory]
+        [InlineData("123456 Hello some text", 7, 10, 123456)]
+        [InlineData("2345678 YESSS", 7, 10, 2345678)]
+        [InlineData("78 A", 7, 10, 78)]
+        [InlineData("0x23CACE A very big number", 1, 16, 2345678)]
```

Comment:
i'm pretty sure `strtol` can't handle the `0x` prefix on hex digits. It would return 0 in this case since x stops the parsing. Let me verify though and I'll chime back in on this thread.

Reply:
Scratch that, the docs indicate `0x` is valid, at least for radix 16. 

Also of note is that this function accepts whitespace up front, as well as leading `-` or `+` signs, so add some tests that cover those cases as well